pkg/engine/combat/heal.go

```diff
@@ -3,32 +3,66 @@ package combat
 import (
 	"github.com/simimpact/srsim/pkg/engine/event"
 	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/model"
 )
 
 func (mgr *Manager) Heal(heal info.Heal) {
+	source := mgr.attr.Stats(heal.Source)
 	for _, t := range heal.Targets {
+		target := mgr.attr.Stats(t)
+
+		// make copy of healMap
 		baseHeal := make(info.HealMap, len(heal.BaseHeal))
 		for k, v := range heal.BaseHeal {
 			baseHeal[k] = v
 		}
 
 		e := &event.HealStart{
-			Target:      mgr.attr.Stats(t),
-			Healer:      mgr.attr.Stats(heal.Source),
+			Target:      target,
+			Healer:      source,
 			BaseHeal:    baseHeal,
 			HealValue:   heal.HealValue,
 			UseSnapshot: heal.UseSnapshot,
 		}
 		mgr.event.HealStart.Emit(e)
 
 		// TODO: Perform Heal. Use the data in the event to perform the heal
-		// TOOD: call ModifyHP to add the new HP to the healed target
+		// Get base heal amount
+		hpLost := target.MaxHP() - target.HP()
+		base := 0.0
```

Comment:
missing the `HealValue` being added to the base similar to how it works with `DamageValue` in `hit`